docs/learn/getting-started.md

```diff
@@ -0,0 +1,123 @@
+---
+sidebar_label: '👉 Getting Started'
+description: Introduction to the LUKSO Ecosystem - why choose LUKSO? Who is LUKSO intended for?
+---
+
+import CallToActionButton from '@site/src/components/CallToActionButton';
+
+# Welcome to LUKSO
+
+The purpose of this section is to teach you the bare minimum, so that you can start coding as soon as possible to try out everything LUKSO has to offer.
+
+For more detailed information on any topic please use search or check out our [LEARN section](../learn/introduction.md).
+
+## What is LUKSO?
+
+LUKSO is layer 1, Ethereum twin blockchain. It uses a proof-of-stake consensus algorithm, making it fundamentally the same as Ethereum at a network level but with some innovative differences, particularly in how developers build applications on the network and the building blocks you use.
```

Comment:
Suggest removing this sentence, as it adds unecessary information (developers building on EVM chains know that Ethereum is PoS)

```suggestion
LUKSO is a Layer 1 EVM chain, Ethereum twin blockchain. It is fundamentally the same as Ethereum at a network level but with some innovative differences, particularly in how developers build applications on the network and the building blocks you use.
```